overlays/prod/embargo16-prod/cnpg-butler-embargo16-pooler.yaml

```diff
@@ -98,4 +98,63 @@ spec:
                 values:
                 - pooler
             topologyKey: "kubernetes.io/hostname"
----
\ No newline at end of file
+---
+apiVersion: postgresql.cnpg.io/v1
+kind: Pooler
+metadata:
+  annotations:
+  name: usdf-butler-embargo16-pooler-transaction-ro
+  namespace: embargo16-prod
+spec:
+  cluster:
+    name: usdf-butler-embargo16
+  instances: 3
+  pgbouncer:
+    parameters:
+      default_pool_size: "1000"
+      idle_transaction_timeout: "0"
+      log_connections: "1"
+      log_disconnections: "1"
+      max_client_conn: "2000"
+      server_idle_timeout: "0"
```

Comment:
Set `server_idle_timeout` to `60`.    During prompt processing connections would hang until this setting was added.